html/popup.html

```diff
@@ -15,7 +15,7 @@
         <p id="start-stream"></p>
     </div>
     <div>
-        <i>Tips: You can hide the player with Alt+Maj+T</i>
+        <i>Tips: You can hide the player with Ctrl+Shift+Z</i>
```

Comment:
Yes, I already checked, and I changed the readme as well. I had to change because the "Alt" key didn't seem to be triggered at all. But I'm open for any change.

Reply:
CTRL+Shift+Z looks too much like the redo command. Are you sure it's not working with the old binds ?

Reply:
Agreed. Totaly sure the old binding does not work. I suspect the problem tu be the "Alt" key. How about "Ctrl+Shift+Space" ?

Reply:
@ImFlog Have you had time to take a look at it?